src/backend/TrafficCourts/Workflow.Service/Consumers/DisputeUpdateRequestAcceptedConsumer.cs

```diff
@@ -1,139 +1,148 @@
-            if (dispute.EmailAddressVerified)
-            {
-                PublishEmailConfirmation(dispute, context);
-            }
-
-            // populate file history
-            PublishFileHistoryLog(dispute, context);
-        }
-    }
-
-    private async void PublishEmailConfirmation(Dispute dispute, ConsumeContext<DisputeUpdateRequestAccepted> context)
-    {
-        SendDispuantEmail message = new()
-        {
-            Message = _updateRequestAcceptedTemplate.Create(dispute),
-            NoticeOfDisputeGuid = new Guid(dispute.NoticeOfDisputeGuid),
-            TicketNumber = dispute.TicketNumber
-        };
-        await context.PublishWithLog(_logger, message, context.CancellationToken);
-    }
-
-    private async void PublishFileHistoryLog(Dispute dispute, ConsumeContext<DisputeUpdateRequestAccepted> context)
-    {
-        SaveFileHistoryRecord fileHistoryRecord = new()
-        {
-            TicketNumber = dispute.TicketNumber,
-            Description = "Disputant update request accepted."
-        };
-        await context.PublishWithLog(_logger, fileHistoryRecord, context.CancellationToken);
-    }
-}
+﻿using MassTransit;
+using Newtonsoft.Json;
+using TrafficCourts.Common.Features.Mail.Templates;
+using TrafficCourts.Common.OpenAPIs.OracleDataApi.v1_0;
+using TrafficCourts.Messaging.MessageContracts;
+using TrafficCourts.Workflow.Service.Services;
+using DisputeUpdateRequest = TrafficCourts.Common.OpenAPIs.OracleDataApi.v1_0.DisputeUpdateRequest;
+
+namespace TrafficCourts.Workflow.Service.Consumers;
+
+public class DisputeUpdateRequestAcceptedConsumer : IConsumer<DisputeUpdateRequestAccepted>
+{
+    private readonly ILogger<DisputeUpdateRequestAcceptedConsumer> _logger;
+    private readonly IOracleDataApiService _oracleDataApiService;
+    private readonly IDisputeUpdateRequestAcceptedTemplate _updateRequestAcceptedTemplate;
+
+    public DisputeUpdateRequestAcceptedConsumer(
+        ILogger<DisputeUpdateRequestAcceptedConsumer> logger,
+        IOracleDataApiService oracleDataApiService,
+        IDisputeUpdateRequestAcceptedTemplate updateRequestAcceptedTemplate)
+    {
+        _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        _oracleDataApiService = oracleDataApiService ?? throw new ArgumentNullException(nameof(oracleDataApiService));
+        _updateRequestAcceptedTemplate = updateRequestAcceptedTemplate ?? throw new ArgumentNullException(nameof(updateRequestAcceptedTemplate));
+    }
+
+    public async Task Consume(ConsumeContext<DisputeUpdateRequestAccepted> context)
+    {
+        // TCVP-1975
+        // - call oracle-data-api to patch the Dispute with the DisputeUpdateRequest changes.
+        // - call oracle-data-api to update DisputeUpdateRequest status.
+        // - send confirmation email indicating request was accepted
+        // - populate file/email history records
+
+        _logger.LogDebug("Consuming message");

```

Comment:
add `using var loggingScope = _logger.BeginConsumeScope(context);` at the start of the consume method to ensure common properties from the context are added to all the log messages in the consumer.

Reply:
thanks -- adding to method